specs/slider/Spec.md

```diff
@@ -32,167 +43,177 @@ Sliders are great for adjusting settings that reflect intensity levels (volume,
 
 
-### Mouse
 
-| Event             | Action                                   | NOTE                                     |
-| ----------------- | ---------------------------------------- | ---------------------------------------- |
-| hover             | highlight slider (both bar & handle)     | Event triggers on both bar & handle hover |
-| click & drag      | moves handle one step forward / backwards | drag right/up -> increase value  \| drag down/left -> decrease value |
-| click (on handle) | highlights handle                        | -                                        |
-| click (on bar)    | moves handle to position where user clicked | -                                        |
+## Accessibility
 
+##### Keyboard
 
+Accessibility for slider is mostly covered with keyboard behavior (according to ARIA docs).
+See [keyboard](#keyboard) section for reference.
 
-### Touch
+##### Focus
 
-| Event           | Action                                   | NOTE                                     |
-| --------------- | ---------------------------------------- | ---------------------------------------- |
-| tap (on handle) | highlights handle                        | we need the ability to expand clickable area for mobile devices |
-| tap (on bar)    | moves handle to position where user clicked | -                                        |
-| drag            | moves handle according to drag           | -                                        |
+Focus is placed on the slider (the visual object that the mouse user would move, also known as the thumb).
 
-NOTE: 
-Later phases of touch handling are going to be implemented using mix in solution that Amir is working on (comment from Gilad).
+> NOTE:
+> Focus is placed on handle according to ARIA doc
+> https://www.w3.org/TR/wai-aria-practices/#slider
 
+![sliderFocus](./assets/sliderFocus.png)
 
-###### Links to ARIA compliant sliders (for reference): 
+**Roles & Attributes**
 
-http://files.paciellogroup.com/blogmisc/ARIA/slider/
-https://www.w3.org/TR/wai-aria-practices/examples/slider/slider-2.html
+| Role   | Attribute               | Element | Usage                                    |
+| ------ | ----------------------- | ------- | ---------------------------------------- |
+| slider |                         | div     | 1) Identifies the element as a slider. 2) Set on the `div` that represents as the movable thumb because it is the operable element that represents the slider value. |
+|        | `tabindex=0`            | `div`   | Includes the slider thumb in the page tab sequence. |
+|        | `aria-valuemax=255`     | `div`   | Specifies the maximum value of the slider. |
+|        | `aria-valuemin=0`       | `div`   | Specifies the minimum value of the slide |
+|        | `aria-valuenow=NUMBER`  | `div`   | Indicates the current value of the slider. |
+|        | `aria-labelledby=IDREF` | `div`   | Refers to the element containing the name of the slider. |
```

Comment:
aria-label, aria-labelledby, aria-describedby should be added to all interactive components.